.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "env",
```

Comment:
env should be specified as node.js envirnoment to avoid redundant transpilation of features that already supported in node (by default it behaves as es6 latest) https://babeljs.io/docs/plugins/preset-env/